setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import find_packages, setup
 
 __package_name__ = "butterfree"
-__version__ = "1.1.1"
+__version__ = "1.1.3"
```

Comment:
why are we skipping 1.1.2?

Reply:
According to @moromimay there is already a 1.1.2, which was a hotfix. Apparently master didn't change because of how releases used to work(I don't know the details)